docs/src/hal/basic-hal.adoc

```diff
@@ -252,7 +250,7 @@ The following commands are depreciated and may be removed from future
 versions. Any new configuration should use the <<sub:hal-net,'net'>> command.
 These commands are included so older configurations will still work.
 
-.linksp
+==== linksp
```

Comment:
>"asciidoc: WARNING: basic-hal.adoc: line 224: missing section: [sect5]"
>
>I think that this rather opaque message means that the docs have too many levels of sub-sections.

Weird that the line number don't match. But should be this level of indentation, right?
These files are also affected:
- axis.adoc
- gmoccapy.adoc
- qtvcp-libraries.adoc
- qtvcp-custom-widgets.adoc
- qtvcp-development.adoc
- classic-ladder.adoc
- basic-hal.adoc

Reply:
I do not see any immediate need to address this prior to "the merge". Once we have our Asciidoc Documentation Style Guide this will likely drag quite a number of changes and these messages can be addressed at the same time.

Reply:
I only was concerned about this comment:

>You can have an asciidoc file that builds OK standalone, but then fails when it is assembled into the full docs, as it then becomes a subsection of the main docs.